### PR TITLE
Bra 12 homework 02

### PR DESCRIPTION
**Pull request by dev-kucha**
Homework 12
Part 2

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    overflow: hidden;
+}
+
+.login-form__label {
+    display: block;
+    color: #fbf6e4;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    font-style: normal;
```
Comment (roskyba) [inline css/styles.css]: `normal` rules, remove them, please

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+    list-style-type: none;
+}
+
+.popup-addlinks__item {
+    color: #b9b9cc;
+    display: inline-block;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 13px;
+    font-weight: 400;
+    font-style: normal;
```
Comment (roskyba) [inline css/styles.css]: and here

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+            box-sizing: border-box;
+    height: 42px;
+    width: 100%;
+    border: 2px solid rgba(191, 185, 204, 0.54);
+    background-color: rgba(0, 0, 0, 0);
+    color: #fbf6e4;
+    font-family: 'Quicksand', sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    font-style: normal;
```
Comment (roskyba) [inline css/styles.css]: and another one and so on :)

**Comment by dev-kucha** [comment]
Everything is fixed. Please check again

**Review by roskyba — APPROVED** [review]

**Pull request closed without merging**